packages/api/src/external/carequality/command/cq-directory/parse-cq-directory-entry.ts

```diff
@@ -211,6 +212,7 @@ function getUrlType(value: string | undefined): ChannelUrl | undefined {
   if (value.includes(XCA_DR_STRING)) return XCA_DR_STRING;
 
   if (value.includes("Direct Messaging")) return;
+  if (value.includes(XDR_STRING)) return; // TODO: Learn about the function of this endpoint and see whether we need to include it in our mapping
```

Comment:
need a ticket for todos (I keep forgetting too)

Reply:
added